lib/pharos/host/el7/scripts/configure-essentials.sh

```diff
@@ -12,6 +12,9 @@ yum_install_with_lock() {
     version=$2
     linefromfile "^0:${package}-" $versionlock
     yum install -y "${package}-${version}"
+    if ! rpm -qi "${package}-${version}" > /dev/null ; then
```

Comment:
I do not get how this works 🤔 Does `yum install ...` just fail silently if trying to install older package version?

Reply:
Yes.